notebooks/api/0.8/11-container-images-k8s.ipynb

```diff
@@ -92,6 +92,16 @@
     "domain_client.worker_pools"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "de9872be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain_client.api.services.worker_pool.scale(number=2, pool_name=\"default-pool\")"
```

Comment:
MAYBE: Could we also do a scale_down check
and we could also add an additional check to see if the number of SyftWorkers in the worker pool match the ones we up/down scaled.